.travis.yml

```diff
@@ -37,8 +37,10 @@ install:
 
 script:
   - cd tests
-  - docker-compose config --services | grep $PLATFORM | xargs docker-compose up -d
-  - ansible-playbook $PLAYBOOK -l $PLATFORM
+  - docker-compose config --services | grep "$PLATFORM" | xargs docker-compose up -d
+  - ansible-playbook $PLAYBOOK -l "$PLATFORM:&latest" --check --diff
+  - ansible-playbook $PLAYBOOK -l "${PLATFORM:-latest}" -vv
```

Comment:
why specify a default value for `$PLATFORM` here? sorry if I misunderstood travis ci workflow, havent used it before, first time looking at configs

Reply:
👍🏿 Good catch ❤️  
This actually a leftover coming from another feature/approach I was working on. I'll remove this default as the PLATFORM variable is expected to be systematically defined (for the moment at least).